Xamarin.Forms.Core/Image.cs

```diff
@@ -76,36 +75,9 @@ public void SetIsLoading(bool isLoading)
 			return _platformConfigurationRegistry.Value.On<T>();
 		}
 
-		BindableProperty IImageController.SourceProperty => SourceProperty;
-		BindableProperty IImageController.AspectProperty => AspectProperty;
-		BindableProperty IImageController.IsOpaqueProperty => IsOpaqueProperty;
+		void IImageElement.OnImageSourcesSourceChanged(object sender, EventArgs e) =>
+			ImageElement.ImageSourcesSourceChanged(this, EventArgs.Empty);
```

Comment:
why are you swallowing the EventArgs ?

Reply:
It was awhile ago but probably because I just copied the pattern from

```

		public static readonly BindableProperty CommandParameterProperty = BindableProperty.Create("CommandParameter", typeof(object), typeof(Button), null,
			propertyChanged: (bindable, oldvalue, newvalue) => ((Button)bindable).CommandCanExecuteChanged(bindable, EventArgs.Empty));
```

I'll change it to not eat it up